src/main/java/fitr/command/EditEntryCommand.java

```diff
@@ -75,8 +75,10 @@ public void execute(ListManager listManager, StorageManager storageManager, User
                 Ui.printInvalidCommandError();
                 break;
             }
-        } catch (NumberFormatException | FitrException | DateTimeParseException e) {
+        } catch (NumberFormatException | FitrException e) {
```

Comment:
May want to check if the number is negative or a letter  in the individual method instead of using NumberFormatException, this may throw "Invalid value entered" instead of "Calories must be between 0 and 10000 (inclusive)!" if the value is too large but still a positive integer.

Reply:
Can provide any sample input that causes that?

Reply:
Won't values like 1873872378291 be too large and throw NumberFormatException?

Reply:
Its larger than the range of values of int so wouldn't it be considered invalid?